src/openrct2/windows/EditorObjectSelection.cpp

```diff
@@ -1742,20 +1739,15 @@ static void window_editor_object_selection_textinput(rct_window *w, rct_widgetin
     window_invalidate(w);
 }
 
-static bool filter_selected(uint8 objectFlag)
+static bool filter_selected(uint8 objectFlags)
```

Comment:
In my opinion the readability of this function hasn't improved

Reply:
Right. But the parameters were named differently in the declaration and implementation and I had to change one of them.

Reply:
I was referring to both conditions in the function, not the argument name.